### PR TITLE
Enable browsingTopics protection

### PR DESCRIPTION
**Pull request by SlayterDev**
**Reviewer:** @jonathanKingston, @sammacbeth 

## Description:

https://app.asana.com/0/1163321984198618/1201719300245228/f

`browsingTopics` is effectively replacing `floc`. This PR turns our `floc` protection into a `browsingTopics` protection.

## Steps to test this PR:
Browsing topics is not yet deployed in current versions of chrome and we're basing this protection on currently available documentation ([see here](https://blog.google/products/chrome/get-know-new-topics-api-privacy-sandbox/)). The content script can be verified by adding a log line to ensure the script is properly injected.

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: README.md
Comment (kdzwinel) [inline README.md:21]: I think we can drop "FLoC" now that browsing-topics effectively replaced it.

File: shared/data/bundled/extension-config.json
Comment (kdzwinel) [inline shared/data/bundled/extension-config.json:3]: IMO we should not mix config updates with unrelated PRs, but rather have one PR that updates the config just before the release.

File: shared/js/background/events.es6.js
Comment (kdzwinel) [inline shared/js/background/events.es6.js:189]: oh, we overlooked isFeatureEnabled before - does that mean that there was no way to disable it via config?

**Review by kdzwinel — COMMENTED** [review]
Nice work @SlayterDev ! I left couple of comments. Also I understand that this is blocked until we merge https://github.com/duckduckgo/content-scope-scripts/pull/17/files ? We should also wrap up internal discussion and figure out how to test it before we merge.

File: shared/js/background/events.es6.js
Comment (jonathanKingston) [reply shared/js/background/events.es6.js:189]: Only for the header yeah.

File: shared/data/bundled/extension-config.json
Comment (jonathanKingston) [reply shared/data/bundled/extension-config.json:3]: I think we'd want this tbf as the key has changed. We could probably mark this file as a binary as it's not really that meaningful anyway.

Comment (SlayterDev) [reply shared/data/bundled/extension-config.json:3]: Agreed. Using our current flow for testing config changes this updated the embedded config when I removed the test version. When we're ready to ship we'll use the version that includes this feature.

File: shared/js/background/events.es6.js
Comment (SlayterDev) [reply shared/js/background/events.es6.js:189]: Specifically the header piece of this. The Content scope script would've been disabled as expected.

**Review by jonathanKingston — CHANGES_REQUESTED** [review]
I think we're missing some changes downstream but everything appears well with the approach.

I think we should not point to the commit in PRs like we do elsewhere, happy to discuss but I think it will make PRs easier etc and just state to the reviewer they need to use `npm link @duckduckgo/content-scope-scripts` for example.

File: shared/data/etags.json
Comment (SlayterDev) [reply shared/data/etags.json:1]: I checked out the version from develop and for some reason it still added a newline.

**Comment by jonathanKingston** [comment]
Just going to wait until we can cut a tag on content scope scripts but other than that we should be good here to merge :)

File: shared/data/etags.json
Comment (jonathanKingston) [reply shared/data/etags.json:1]: Can you just run:
```
git checkout origin/develop -- shared/data/etags.json
git checkout origin/develop -- package-lock.json
```

Neither should be needed here and we rely on just updating etags for the release as it changes so often.

**Review by jonathanKingston — APPROVED** [review]

**Pull request merged**
